### PR TITLE
Add cpu random gen and tests

### LINKED ISSUE #368: Test data generator
Create a new instance of CPU with random numbers for testing.
```rs
 CPU {
    cur_freq: -1,
    cur_usage: 345.0,
    cur_temp: 453,
    gov: "gov".to_string(),
    max_freq: -1,
    min_freq: -1,
    name: "dsf".to_string(),
    number: 0,
},
```

### PR DESCRIPTION
**Pull request by JakeRoggenbuck**
fixes #368

### CODE REVIEW

File: src/cpu.rs [hunk 1]
```diff
@@ -178,4 +180,36 @@ impl Speed for CPU {
...
+            max_freq: rng.gen_range(0..100000),
+            min_freq: rng.gen_range(0..10000),
+            cur_freq: rng.gen_range(0..100000),
+            cur_temp: rng.gen_range(0..100000),
+            cur_usage: rng.gen::<f32>(),
+            gov: if rng.gen_bool(0.5) {
+                "powersave".to_string()
+            } else {
+                "performance".to_string()
+            },
```
Comment (Shuzhengz) [inline src/cpu.rs:198]: o are we not gonna test schutil?

**Review by Shuzhengz — APPROVED** [review]
lgtm

File: src/cpu.rs [hunk 2]
```diff
@@ -178,4 +180,36 @@ impl Speed for CPU {
...
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn cpu_random_unit_test() {
+        let cpu_1 = CPU::random();
+        let cpu_2 = CPU::random();
+
+        assert_ne!(cpu_1.cur_temp, cpu_2.cur_temp);
```
Comment (Camerooooon) [inline src/cpu.rs:212]: I don't understand this test, what happens if both cpus roll the same number? What is the point of filling it with random information.

Comment (Camerooooon) [reply src/cpu.rs:212]: @JakeRoggenbuck ^

Comment (Shuzhengz) [reply src/cpu.rs:212]: @JakeRoggenbuck

**Pull request merged**